main/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
-idf_component_register(SRCS "main.c"
-                    INCLUDE_DIRS ".")
-                    
+idf_component_register(SRCS "main.c" "GlobalInit.c"
+                    INCLUDE_DIRS "include" 
+                    # REQUIRES SpotifyAuthorization nvs_flash WiFiComponent HttpsCoponent_ nvsFlash 
```

Comment:
every component add to make component automaticly and dont need to REQUIRE them.

Reply:
ok